source/custom/nixnet_service.custom.cpp

```diff
@@ -1,2 +1,207 @@
+            case nixnet_grpc::ReadState::READ_STATE_STATE_TIME_COMMUNICATING :{
+                response->mutable_state_value()->set_time_communicating(*((u64*)state_value));
+                break;
+            } 
+            case nixnet_grpc::ReadState::READ_STATE_STATE_TIME_START :{
+                response->mutable_state_value()->set_time_start(*((u64*)state_value));
+                break;
+            }
+            case nixnet_grpc::ReadState::READ_STATE_STATE_CAN_COMM :{
+                response->mutable_state_value()->set_can_comm(*((u32*)state_value));
+                break;
+            }
+            case nixnet_grpc::ReadState::READ_STATE_STATE_FLEX_RAY_COMM :{
+                response->mutable_state_value()->set_flex_ray_comm(*((u32*)state_value));
+                break;
+            }
+            case nixnet_grpc::ReadState::READ_STATE_STATE_LIN_COMM :{
+                response->mutable_state_value()->set_lin_comm(*((u32*)state_value));
+                break;
+            }
+            case nixnet_grpc::ReadState::READ_STATE_STATE_SESSION_INFO :{
+                response->mutable_state_value()->set_session_info(*((u32*)state_value));
+                break;
+            }
+            case nixnet_grpc::ReadState::READ_STATE_STATE_FLEX_RAY_STATS :{
+                convert_to_grpc(*(_nxFlexRayStats_t*)state_value, response->mutable_state_value()->mutable_flex_ray_stat());
+                break;
+            }
+            case nixnet_grpc::ReadState::READ_STATE_STATE_J1939_COMM :{
+                convert_to_grpc(*(_nxJ1939CommState_t*)state_value, response->mutable_state_value()->mutable_comm_stat());
+                break;
+            }
+            case nixnet_grpc::ReadState::READ_STATE_STATE_TIME_CURRENT_2 :{
+                convert_to_grpc(*(_nxTimeLocalNetwork_t*)state_value, response->mutable_state_value()->mutable_time_current2());
+                break;
+            }
+            case nixnet_grpc::ReadState::READ_STATE_STATE_TIME_COMMUNICATING_2 :{
+                convert_to_grpc(*(_nxTimeLocalNetwork_t*)state_value, response->mutable_state_value()->mutable_time_communicating2());
+                break;
+            } 
+            case nixnet_grpc::ReadState::READ_STATE_STATE_TIME_START_2 :{
+                convert_to_grpc(*(_nxTimeLocalNetwork_t*)state_value, response->mutable_state_value()->mutable_time_start2());
+                break;
+            }
+        }
+            response->set_fault(fault);
+        }
+        return ::grpc::Status::OK;
+    }
+    catch (nidevice_grpc::LibraryLoadException& ex) {
+        return ::grpc::Status(::grpc::NOT_FOUND, ex.what());
+    }
+}
+
+//---------------------------------------------------------------------
+//---------------------------------------------------------------------
+::grpc::Status NiXnetService::ReadStateTimeTrigger(::grpc::ServerContext* context, const ReadStateTimeTriggerRequest* request, ReadStateTimeTriggerResponse* response)
```

Comment:
Why isn't this method generated? This seems pretty standard.

Reply:
Auto generated this method